hydra-node/src/Hydra/Ledger/Cardano/Builder.hs

```diff
@@ -55,23 +55,24 @@ instance Exception InvalidTransactionException
 emptyTxBody :: TxBodyContent BuildTx
 emptyTxBody =
   TxBodyContent
-    mempty -- inputs
-    (TxInsCollateral mempty) -- FIXME
-    TxInsReferenceNone
-    mempty -- outputs
-    TxTotalCollateralNone
-    TxReturnCollateralNone
-    (TxFeeExplicit 0)
-    (TxValidityNoLowerBound, TxValidityNoUpperBound)
-    TxMetadataNone
-    TxAuxScriptsNone
-    TxExtraKeyWitnessesNone
-    (BuildTxWith $ Just $ fromLedgerPParams ShelleyBasedEraBabbage def) -- FIXME
-    TxWithdrawalsNone
-    TxCertificatesNone
-    TxUpdateProposalNone
-    TxMintValueNone
-    TxScriptValidityNone
+    { txIns = mempty -- inputs
+    , txInsCollateral = TxInsCollateral mempty -- FIXME
+    , txInsReference = TxInsReferenceNone
+    , txOuts = mempty -- outputs
+    , txTotalCollateral = TxTotalCollateralNone
+    , txReturnCollateral = TxReturnCollateralNone
+    , txFee = TxFeeExplicit 0
+    , txValidityRange = (TxValidityNoLowerBound, TxValidityNoUpperBound)
+    , txMetadata = TxMetadataNone
+    , txAuxScripts = TxAuxScriptsNone
+    , txExtraKeyWits = TxExtraKeyWitnessesNone
+    , txProtocolParams = BuildTxWith $ Just $ fromLedgerPParams ShelleyBasedEraBabbage def -- FIXME
+    , txWithdrawals = TxWithdrawalsNone
+    , txCertificates = TxCertificatesNone
+    , txUpdateProposal = TxUpdateProposalNone
+    , txMintValue = TxMintValueNone
+    , txScriptValidity = TxScriptValidityNone
+    }
```

Comment:
Thanks, but would have not been required in this PR :)

Reply:
I know, but is smth i've been wanting to push for a long time :P